packages/dev/tsconfig/CHANGELOG.md

```
@@ -0,0 +1,6 @@
+## [3.0.0] - 2025-09-04
+
+### ⚠ Breaking Changes
+- Updated `module` setting to `Node20` in `tsc` and `tsc-dom` configs.
+    - Requires **TypeScript >= 5.9.0**.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)